docs/core/tools/csproj.md

```diff
@@ -69,6 +69,13 @@ This change does not modify the main mechanics of other includes. However, if yo
 ### Recommendation
 With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
 
+## How to see the whole project as MSBuild sees it
+
+While those csproj changes greatly simplify project files, it is sometimes useful to be able to look at the fully expanded project as MSBuild sees it once the SDK and its targets are included. This can be done by simply typing the following in the command-line:
```

Comment:
it is sometimes useful to be able to look at the -> can you simplify this sentence? perhaps "you might wanna see the"

This can be done by simply typing the following in the command-line -> This can be done by specifying the `/pp` switch to the [dotnet msbuild](dotnet-msbuild.md) command, such as in the following example:

My suggestion is based on the fact that you can also pass a filepath to the /pp switch, no? (although I'm having some issues using dotnet msbuild /pp:[filepath]. It's doing some weird stuff. I might file a CLI bug on this once I confirm the behavior.